build.sh

```diff
@@ -76,8 +76,10 @@ export TARGET TARGET_FAMILY TARGET_SUBFAMILY TARGET_PROJECT PROJECT_PATH PREFIX_
 EXPORT_CFLAGS="$(make -f phoenix-rtos-build/Makefile.common export-cflags)"
 # Convert ldflags to format recognizable by gcc, for example -q -> -Wl,-q
```

Comment:
you can remove this comment as it's obsolete now (we're linking internally by gcc so no conversion is needed here)

Reply:
Done